packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/TorDropdown/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { Translation } from '@suite-components';
+import { Dropdown, DropdownRef, Icon, useTheme, variables } from '@trezor/components';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import { transparentize } from 'polished';
+import * as routerActions from '@suite-actions/routerActions';
+
+const Wrapper = styled.div<Pick<Props, 'marginLeft'>>`
+    ${props => props.marginLeft && `margin-left: 8px`};
+    position: relative;
+`;
+
+const TorStatus = styled.div`
+    display: flex;
+    cursor: default;
+    align-items: center;
+    & > * + * {
+        margin-left: 10px;
+    }
+`;
+
+const Indicator = styled.div<{ isActive?: boolean }>`
+    display: flex;
+    width: 24px;
+    height: 24px;
+    border-radius: 50%;
+    position: relative;
+    background: ${props =>
+        props.isActive
+            ? transparentize(0.9, props.theme.TYPE_GREEN)
+            : transparentize(0.9, props.theme.TYPE_LIGHT_GREY)};
```

Comment:
Do we really need to add library for these two lines?

We can use 8-digit hex where the two last chars are for opacity.

```suggestion
            ? `${props.theme.TYPE_GREEN}19`
            : `${props.theme.TYPE_LIGHT_GREY}19`
```

19 is same as 10% opacity

I have to admit that it looks confusing but it should do the job. Probably creating some style helper for that would be nice!

Reply:
As per our discussion on Friday's standup, I'd like to keep the library in the project and I'd like to utilise other helpers from the library too. The helpers are exported from the package as a standalone functions and should be properly tree-shakable, so they shouldn't introduce much overhead with a proper build setup.